config/routes.php

```diff
@@ -200,9 +200,14 @@
         ['pass' => ['id', 'relation'], '_name' => 'modules:resourcesJson']
     );
     $routes->connect(
-        '/:object_type/view/:id/treeJson/:relation',
+        '/:object_type/view/:id/treeJson',
```

Comment:
you could keep only the first rule and check `root` as optional query string in `treeJson` method

[depending on how you generate routes...]